src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,321 @@
+                taskType = "event";
+            } else if (isDeadline) {
+                tasks.add(new Deadline(command.replace("deadline ", "")));
+                taskType = "deadline";
+            }
+
+            addCompleteMessage(taskType);
+            appendToFile(taskType);
+
+        } catch (EmptyDescriptionException emptyException) {
+            System.out.println("OOPS!!! The description of a todo cannot be empty.");
+        } catch (IOException e) {
+            System.out.println("Error writing to file.");
+        } catch (NoSpecficTimeException e) {
+            System.out.println("No time is entered. Please re-enter and include '/at'.");
+        } catch (NoSpecificDeadlineException e) {
+            System.out.println("No deadline is entered. Please re-enter and include '/by'. ");
+        }
+
+    }
+
+    public static void reAddTask(String command) {
+        boolean isTodo = command.contains("todo");
+        boolean isEvent = command.contains("event");
+        boolean isDeadline = command.contains("deadline");
+
+        try {
+            if (isTodo) {
+                tasks.add(new Todo(command.replace("todo", "").trim()));
+            } else if (isEvent) {
+                tasks.add(new Event(command.replace("event", "").trim()));
+            } else if (isDeadline) {
+                tasks.add(new Deadline(command.replace("deadline", "").trim()));
+            }
+
+        } catch (EmptyDescriptionException emptyException) {
+            System.out.println("OOPS!!! The description cannot be empty.");
+        } catch (NoSpecificDeadlineException e) {
+            System.out.println("You did not enter the deadline. Please re-enter the deadline by including '/by'.");
+        } catch (NoSpecficTimeException e) {
+            System.out.println("You did not specify the time for the event. Please re-enter the event by including '/at'. ");
+        }
+    }
+
+    public static void printDeletedTask(int deleteNumber) {
+        System.out.println("  [T][ ] " + tasks.get(deleteNumber).getTask());
+    }
+
+    public static void removeFromFile(int deleteNumber) throws IOException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        String text = "";
+        int currentIndex = 0;
+        while (s.hasNext()) {
+            String currentLine = s.nextLine();
+            if (deleteNumber != currentIndex) {
+                text += currentLine + System.lineSeparator();
+            }
+            currentIndex += 1;
+        }
+
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(text);
+        fw.close();
+    }
+
+    public static void deleteTask(String command) throws IOException {
+        int deleteNumber = Integer.parseInt(command.split(" ")[1]) - 1;
+        printDeletedTask(deleteNumber);
+        tasks.remove(deleteNumber);
+        removeFromFile(deleteNumber);
+        printNumberOfTask();
+    }
+
+    public static void addCompleteMessage(String type) {
```

Comment:
Is the role of this method to display the message after adding a task? Would be better to rename it to showAddCompletedMessage() if that is the case, to avoid any confusion.